### PR TITLE
Modify: Zen-Team.Zen-Browser version 1.0.0-a.32

### PR DESCRIPTION
**Pull request by UnownPlain**
Checklist for Pull Requests
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/170369)

### CODE REVIEW

**Comment by UnownPlain** [comment]
The website works, the CI fails because it gets stuck on a browser/security check.

**Comment by stephengillie** [comment]
> The website works, the CI fails because it gets stuck on a browser/security check.

Unfortunately, this is common with our automation's frequent checks.

File: manifests/z/Zen-Team/Zen-Browser/Optimized/1.0.0-a.31/Zen-Team.Zen-Browser.Optimized.locale.en-US.yaml [hunk 1]
```diff
@@ -5,25 +5,25 @@ PackageIdentifier: Zen-Team.Zen-Browser.Optimized
 PackageVersion: 1.0.0-a.31
 PackageLocale: en-US
 Publisher: Mozilla
-PublisherUrl: https://github.com/zen-browser
+PublisherUrl: https://www.zen-browser.app
```
Comment (stephengillie) [inline manifests/z/Zen-Team/Zen-Browser/Optimized/1.0.0-a.31/Zen-Team.Zen-Browser.Optimized.locale.en-US.yaml:8]:
```suggestion
# PublisherUrl: 
```

File: manifests/z/Zen-Team/Zen-Browser/Optimized/1.0.0-a.31/Zen-Team.Zen-Browser.Optimized.locale.en-US.yaml [hunk 2]
```diff
@@ -5,25 +5,25 @@ PackageIdentifier: Zen-Team.Zen-Browser.Optimized
...
 PublisherSupportUrl: https://github.com/zen-browser/desktop/issues
 PackageName: Zen Browser (Optimized build)
-PackageUrl: https://github.com/zen-browser/desktop
+PackageUrl: https://www.zen-browser.app
```
Comment (stephengillie) [inline manifests/z/Zen-Team/Zen-Browser/Optimized/1.0.0-a.31/Zen-Team.Zen-Browser.Optimized.locale.en-US.yaml:11]:
```suggestion
# PackageUrl: 
```

**Review by stephengillie — CHANGES_REQUESTED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**